Manager/InMemory/AuthorizationCodeManager.php

```diff
@@ -36,4 +36,15 @@ public function clearExpired(): int
 
         return $count - \count($this->authorizationCodes);
     }
+
+    public function clearRevoked(): int
+    {
+        $count = \count($this->authorizationCodes);
+
+        $this->accessTokens = array_filter($this->authorizationCodes, static function (AuthorizationCode $accessToken): bool {
+            return !$accessToken->isRevoked();
+        });
+
+        return $count - \count($this->accessTokens);
```

Comment:
@franjo-zadelj-trikoder `$this->accessTokens`?

please add tests for all in memory implementations.

all three implementations already have unit tests:


https://github.com/trikoder/oauth2-bundle/tree/v3.x/Tests/Unit